src/include/btmem.h

```diff
@@ -222,18 +222,17 @@ struct __wt_ovfl_reuse {
 #else
 #define WT_LOOKASIDE_COMPRESSOR "none"
 #endif
-#define WT_LAS_CONFIG                                                             \
-    "key_format=" WT_UNCHECKED_STRING(QIQu) ",value_format=" WT_UNCHECKED_STRING( \
-      QQQBBu) ",block_compressor=" WT_LOOKASIDE_COMPRESSOR                        \
-              ",leaf_value_max=64MB"                                              \
+#define WT_LAS_CONFIG                                                            \
+    "key_format=" WT_UNCHECKED_STRING(IuQ) ",value_format=" WT_UNCHECKED_STRING( \
```

Comment:
Not sure. It's not a good time to perf test since some pretty fundamental things are broken in our branch. But I'll revisit this later when we get closer to merging into `develop`.